app/src/main/java/com/android/unio/ui/explore/Explore.kt

```diff
@@ -120,23 +120,29 @@ fun ExploreScreenContent(padding: PaddingValues, navigationAction: NavigationAct
 
 @Composable
 fun AssociationItem(association: Association, navigationAction: NavigationAction) {
-  Column(modifier = Modifier.clickable { navigationAction.navigateTo(Screen.ASSOCIATION) }) {
-    /**
-     * AdEC image is used as the placeholder. Will need to add the actual image later, when the
-     * actual view model is used.
-     */
-    Image(
-        painter = painterResource(id = R.drawable.adec),
-        contentDescription = "image description",
-        modifier = Modifier.size(124.dp))
+  Column(
+      modifier =
+          Modifier.clickable {
+            navigationAction.navigateTo(
+                Screen.ASSOCIATION_PROFILE +
+                    "/{uid}".replace(oldValue = "{uid}", newValue = association.uid))
```

Comment:
I'd consider extracting the string replacement logic in a separate function to reduce cognitive complexity and enable reusability, as navigation route parameters will likely be used in other parts of the project.

Reply:
Also, I'm noticing this could be simplified to just `"${Screen.ASSOCIATION_PROFILE}/${association.uid}"`

Reply:
> I'd consider extracting the string replacement logic in a separate function to reduce cognitive complexity and enable reusability, as navigation route parameters will likely be used in other parts of the project.

I would argue that having a separate function is really useful, as I don't see how to both generalize it and make it easy to the eye in terms of what arguments would be needed for the function, since it is possible that other parts of the project would use multiple arguments. 
Maybe I don't see it how you do, how would you define a helper function for this replacement?

> Also, I'm noticing this could be simplified to just `"${Screen.ASSOCIATION_PROFILE}/${association.uid}"`

That's true!